critiquebrainz/frontend/templates/review/browse.html

```diff
@@ -5,6 +5,16 @@
 
 {% block content %}
 <h2>{{ _('Reviews') }}</h2>
+<form method="GET" action="{{ url_for('review.browse') }}">
+  <div class="form-group" style="display: inline-block;">
+    <select class="form-control input-sm" name="entity_type" onchange="this.form.submit()"> 
```

Comment:
I did not include 'Entity type:'  because it broke the line and looked congested on smaller screens.
Do we need to do that? I think, with all the entity types listed it's quite self-explanatory.